update_compilers/install_compilers.sh

```diff
@@ -235,6 +235,9 @@ install_new_rust 1.27.0 RUST_TARGETS[@]
 install_new_rust 1.27.1 RUST_TARGETS[@]
 install_new_rust 1.28.0 RUST_TARGETS[@]
 if install_nightly; then
+    RUST_TARGETS+=(
```

Comment:
Wow, I didn't know you could do that in `bash` :)

Reply:
@mjbshaw showed it to me here, I didn't know that either. Now I have to look what all those `@` are.

Reply:
The `@`s are used to pass the array as an array. Bash supports "true" arrays (which is what you get when you use parens (i.e. `(foo bar baz)`) or `local -a`), and in order to pass the whole array as a single argument (rather than splatting the array) you have to use the `[@]` syntax.

At least that's my understanding. I could be wrong. My Bash-foo is actually pretty weak. I discovered this when writing my first patch and reading up on how to deal with Bash's arrays (using [Bash Guide for Beginners](http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_10_02.html) and the [GNU Bash Manual](https://www.gnu.org/software/bash/manual/html_node/Arrays.html)).

Reply:
Thanks @mjbshaw for the explanation :)